keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,321 @@
+This means that there will be an increased risk, which would usually be split
+up into 4 dedicated milestones rather than 3.
+
+SIG Release cannot mitigate this risk directly, but is able to track and
+influence it during each release cycle. It's the responsibility of SIG Release,
+together with SIG Testing and SIG Architecture, to identify new gaps and issues
+in the release cadence and mitigate them on a case-by-case basis.
+
+#### Attention to tests
+
+This KEP does not propose any change to the release cycle itself and assumes
+that the same periods for Code and Test Freeze. Assuming that, there is an
+increased risk for flakes and test failures. It will be the responsibility of
+SIG Release to mitigate this, together with the CI signal role. If we speak
+about an overall release cycle enhancement of 3-4 weeks, then we believe that
+SIG Release is able to mitigate this risk over multiple releases.
+
+#### Attention to dependencies
+
+Having fewer releases will introduce the risk of missing dependencies—for
+example, Golang upgrades. This has to be mitigated on a case-by-case basis, in
+the same way as it is being done right now.
+
+## Design Details
+
+### Schedule Policy
+
+The exact details of the release schedule are created by the team leads before
+the actual cycle begins. SIG Release only provides a lightweight policy, which
+is defined as:
+
+1. The first Kubernetes release of a year should start at the second or third
+   week of January to provide people more room after coming back from the
+   Christmas holidays.
+
+2. The last Kubernetes release of a year should be finished by the middle of
+   December.
+
+3. A Kubernetes release cycle has a length of of ~15 weeks.
+
+4. Events like KubeCon will be considered as blocked from development or
+   decision-making. SIG Release will also consider the week before and after
+   the event in the same way.
+
+5. An explicit break of at least two weeks between each release cycle will be
+   enforced.
+
+   This does not mean that zero development can happen during that time.
+   Rather, SIG Release will use this time to do the release retrospective and
+   plan for the next cycle.
+
+### Feedback survey
+
+Each minor Kubernetes release will be an experience survey, which will include
+questions around the release cadence.
+
+Survey contents are to be determined, but we welcome content suggestions to
+continually improve the process.
+
+Post-release surveys will close after the `.2` patch release to allow the team
+sufficient time to process and incorporate feedback.
+
+Using Kubernetes v1.19 date to provide an example of the survey timeline:
+
+- 2020-08-26: v1.19.0 released (survey would go out)
+- 2020-09-09: v1.19.1 released
+- 2020-09-16: v1.19.2 released (survey would close)
+
+With this example, the survey would have been open for three weeks.
+With an extended release cycle, post-release surveys would be open for around
+three to six weeks (depending on the patch release schedule).
```

Comment:
Why we bind to the `.2` patch release instead of defining a timeline, for example, 1 month after the `.0` release? As seen in the example, this is very unpredictable. This way we can't clearly communicate the deadline, which can be confusing. Users/contributors might not be aware of when the release will be out until it's not actually released, especially if we have out of cadence releases.

Reply:
I think this point is obsolete of we say we do an end user survey every 3 releases.